config/default.yml

```diff
@@ -1172,10 +1172,13 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   CountComments: false  # count full line comments?
   Max: 25
-  Exclude:
-    - 'Rakefile'
-    - '**/*.rake'
-    - 'spec/**/*.rb'
+  ExcludedMethods:
```

Comment:
Guess we should remove the exclusions completely and just modify RuboCop's own config.